docs/02-admin/01-installation/02-docker.md

```diff
@@ -48,83 +45,57 @@ git clone https://github.com/MbinOrg/mbin.git
 cd mbin
 ```
 
-### Docker image preparation
-
-> [!NOTE]
-> If you're using a version of Docker Engine earlier than 23.0, run `export DOCKER_BUILDKIT=1`, prior to building the image. This does not apply to users running Docker Desktop. More info can be found [here](https://docs.docker.com/build/buildkit/#getting-started)
+### Environment configuration
 
-1. First go to the _docker directory_:
+Use either the automatic environment setup script _OR_ manually configure the `.env`, `compose.override.yaml`, and OAuth2 keys. Select one of the two options.
 
-```bash
-cd docker
-```
+> [!TIP]
+> Everything configured for your specific instance is in `.env`, `compose.override.yaml`, and `storage/` (assuming you haven't modified anything else). If you'd like to backup, or even completely reset/delete your instance, then these are the files to do so with.
 
-2. Use the existing Docker image _OR_ build the docker image. Select one of the two options.
-
-#### Build our own Docker image
-
-If you want to build our own image, run (_no_ need to update the `compose.yml` file):
-
-```bash
-docker build --no-cache -t mbin -f Dockerfile  ..
-```
-
-#### Use Mbin pre-build image
+#### Automatic setup
 
-_OR_ use our pre-build images from [ghcr.io](https://ghcr.io). In this case you need to update the `compose.yml` file:
+Run the setup script and pass in a mode (either `prod` or `dev`) and your domain (which can be `localhost` if you plan to just test locally):
 
 ```bash
-nano compose.yml
+./docker/setup.sh prod mbin.domain.tld
 ```
```

Comment:
So let's say I just followed this guide. And I went for the setup script.

 It would be nice to point the user to the next step after existing this command.. Since what now? I see a lot of sections below about manual configuration. 
 
Explain to the user what are the follow up steps and/or point to the next step in the guide / link to the next heading..?

Reply:
Does this look good: https://github.com/MbinOrg/mbin/pull/1542/commits/8ce74401d5319e7a76684d5d098045376e92a414?

Reply:
almost ;)

Reply:
ow nevermind its good now. I thought `> [!WARNING]` was on the same line. That would cause issues, but it seems to work fine now.